meson.build

```diff
@@ -0,0 +1,17 @@
+project('sassc', 'c', version: '3.6.2', default_options: ['c_std=c11'])
+
+if target_machine.system() == 'windows'
+  add_project_arguments('-D_WIN32', language: 'c')
```

Comment:
_WIN32 is predefined on Windows, no need to set it again.

Reply:
Oh, thanks for the heads-up I didn't know.